otter/convergence/service.py

```diff
@@ -114,6 +115,10 @@ def execute_convergence(tenant_id, group_id, log,
     launch_config = manifest['launchConfiguration']
     now = yield Effect(Func(time.time))
 
+    group_status = ScalingGroupStatus.lookupByName(manifest['status'])
+    if group_status == ScalingGroupStatus.DELETING:
+        group_state.desired = 0
```

Comment:
I was wondering about that too, and seeing where group_state was being written or something.  I think that makes more sense, and makes it clearer that we aren't changing anything in the, just deleting the group resources